app/Observers/UserHistoryObserver.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Toby\Observers;
+
+use Laragear\CacheQuery\Facades\CacheQuery;
+use Toby\Models\UserHistory;
+
+class UserHistoryObserver
+{
+    public function creating(UserHistory $userHistory): void
+    {
+        CacheQuery::forget("userHistories{$userHistory->user->id}");
```

Comment:
```suggestion
        CacheQuery::forget("user:history:{$userHistory->user->id}");
```
I think that would be easier to manage.

Reply:
Should I update other cache keys?